src/LinkedList.js

```diff
@@ -0,0 +1,129 @@
+class Node {
+  constructor() {
+    this.data
+    this.next = null
+  }
+}
+
+class LinkedList {
+  constructor() {
+    this.head = null
+    this.tail = null
+  }
+
+  getHeadNode() {
+    return this.head
+  }
+
+  getTailNode() {
+    return this.tail
+  }
+
+  contains(element) {
+    let elementFound = false
+    let iterator = this.head
+    while(iterator) {
+      if(iterator.data === element) elementFound = true
```

Comment:
Instead of using a boolean value, I would just `return true` directly within the `while`loop here.

```javascript

if (iterator.data === element) return true
```